custom_dc/README.md

```diff
@@ -0,0 +1,262 @@
+# Custom Data Commons
+
+A custom Data Commons can hold new data and include UI modifications that are
+custom to a third party's need.
+
+This document illustrates development and deployment details for a custom Data
+Commons instance.
+
+## Prerequisites
+
+- Install [docker](https://www.docker.com/products/docker-desktop/)
+- [Optional] Install [gcloud](https://cloud.google.com/sdk/docs/install-sdk)
+  command line tool.
+- [Optional] Have a Google Cloud Project that can be used to deploy the custom
+  instance.
+
+## API Key
+
+- A custom Data Commons needs to connect with main Data Commons. Get API key for
+  Data Commons by sending an email to `support@datacommons.org`.
+
+- [Optional] Provision a Google Maps API key from your GCP project. This is
+  optional and used for place search in visualization tools. Refer to [Maps API
+  Key](TODO) section for detailed instructions.
+
+## Local Development
+
+Custom Data Commons instance can be developed and tested locally by following
+the instructions below:
+
+### Build docker image locally
+
+```bash
+docker build --tag datacommons-website-compose:latest \
+-f build/web_compose/Dockerfile \
+-t website-compose .
+```
+
+### Run custom Data Commons locally with SQLITE database
+
+In the root of this repository, run:
+
+```bash
+docker run -it \
+--env-file $PWD/custom_dc/sqlite_env.list \
+-p 8080:8080 \
+-v $PWD/custom_dc/sample:/sqlite \
+-v $PWD/server/templates/custom_dc:/workspace/server/templates/custom_dc \
+datacommons-website-compose:latest
+```
+
+This brings up a local instance with sample data that are stored under
+`custom_dc/sample` folder.
+
+Now you can open `localhost:8080/tools/timeline` to browse these sample data.
+Also note the base Data Commons data are also avaiable in this instance.
+
+## Cloud Development and Deployment
+
+Custom Data Commons can be ran on the cloud as a production service. The SQLite
+approach can still be used. Note you need to copy the data folders into the
+cloud disk and mount it to the docker container then specify the enviornment
```

Comment:
Link to something on "cloud disk"? Do you mean https://cloud.google.com/persistent-disk ?

Reply:
Yes, added example of using persistent volume in GKE